<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalculator</class>
 <widget class="QDialog" name="CreditCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>440</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Кредитный калькулятор</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_total_loan_amount">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>3</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	text-align: center
 	border: 1px solid gray;
}

background-color : white;</string>
      </property>
      <property name="text">
       <string>Общая сумма кредита:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="total_loan_amount">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="maximum">
       <number>1000000000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_credit_term">
      <property name="styleSheet">
       <string notr="true">QLabel {
	text-align: center
 	border: 1px solid gray;
}

background-color : white;</string>
      </property>
      <property name="text">
       <string>Срок(в месяцах):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="credit_term">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_credit_term_2">
      <property name="styleSheet">
       <string notr="true">QLabel {
	text-align: center
 	border: 1px solid gray;
}

background-color : white;</string>
      </property>
      <property name="text">
       <string>Ставка(годовая):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="credit_lending_rate">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>227</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton_annuity_payments">
      <property name="text">
       <string>Аннуитетные платежи</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_differentiated_payments">
      <property name="text">
       <string>Дифференцированные платежи</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_overpayment">
      <property name="styleSheet">
       <string notr="true">QLabel {
	text-align: center
 	border: 1px solid gray;
}

background-color : white;</string>
      </property>
      <property name="text">
       <string>Переплата по кредиту:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="overpayment">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_total_payment">
      <property name="styleSheet">
       <string notr="true">QLabel {
	text-align: center
 	border: 1px solid gray;
}

background-color : white;</string>
      </property>
      <property name="text">
       <string>Общая выплата:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="total_payment">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="month_number">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>3</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	text-align: center
 	border: 1px solid gray;
}

background-color : white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_right">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_monthly_payment_diff">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>108</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_left">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_monthly_payment">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>3</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	text-align: center
 	border: 1px solid gray;
}

background-color : white;</string>
   </property>
   <property name="text">
    <string>Ежемесячный платёж:</string>
   </property>
  </widget>
  <widget class="QLabel" name="monthly_payment">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>300</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
